src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -397,39 +421,39 @@ docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock \
+You can also build your own `INIT` and `SERVER` images to have custom configuration and binaries. For this, clone the https://github.com/eclipse/che[Che repo] and copy the `dockerfiles` directory to the root of your custom assembly. If your custom Che server does not need any custom configuration, build the Che server image by executing the `dockerfiles/build.sh` command. Once done, tag the resulted image as needed. If your custom Che server requires custom configuration and you want to let users override them in the `che.env`file, build your own `INIT` image with a custom https://github.com/eclipse/che/blob/master/dockerfiles/init/manifests/che.env[che.env] file.
 
-Once done, you can start your custom binaries this way:
+Start the custom binaries.
 
 ----
 docker run -ti -v '/var/run/docker.sock:/var/run/docker.sock -v /local/data/path:/data -e "IMAGE_CHE=your/che-server" -e "IMAGE_INIT=your/init-image" eclipse/che:$tag start'
 ----
 
-`IMAGE_CHE` is the image you have built in `dockerfiles/che`, and `IMAGE_INIT` is the one from `dockerfiles/init`.
+You have built `IMAGE_CHE` in `dockerfiles/che` and `IMAGE_INIT` is the one from `dockerfiles/init`.
 
 [id="docker-unix-socket-mounting-vs-tcp-mode"]
-== Docker Unix Socket Mounting vs TCP Mode
+== Docker Unix socket mounting vs TCP mode
 
-The `-v /var/run/docker.sock:/var/run/docker.sock` syntax is for mounting a Unix socket so that when a process inside the container speaks to a Docker daemon, the process is redirected to the same socket on the host system.
+The `-v /var/run/docker.sock:/var/run/docker.sock` command is used for mounting a Unix socket so that when a process inside the container communicates with a Docker daemon, the process is redirected to the same socket on the host system.
 
 However, peculiarities of file systems and permissions may make it impossible to invoke Docker processes from inside a container. If this happens, the Che startup scripts will print an error about not being able to reach the Docker daemon with guidance on how to resolve the issue.
 
-An alternative solution is to run Docker daemon in TCP mode on the host and export `DOCKER_HOST` environment variable in the container. You can tell the Docker daemon to listen on both Unix sockets and TCP. On the host running the Docker daemon:
+An alternative solution is to run Docker daemon in the TCP mode on the host and export the `DOCKER_HOST` environment variable in the container. You can make the Docker daemon to listen on both Unix sockets and TCP. On the host running the Docker daemon:
```

Comment:
"run the Docker daemon in TCP mode"

"You can make the Docker daemon listen on both Unix sockets and TCP."